flake.nix

```diff
@@ -110,6 +118,18 @@
 
           stdenv = pkgs.stdenv;
 
+          clightning-dev = pkgs.clightning.overrideAttrs (oldAttrs: {
```

Comment:
**issue (blocking):** This should be removed now and only `pkgs.clightning` used everywhere.

Reply:
Sorry I did not check the rebase result!